### DISCUSSION TITLE
Online accounts support just like Gnome(Or better)

### DISCUSSION THREAD

**Discussion by hackstash** [elementary/switchboard-plug-onlineaccounts#150] [Ideas]
## Problem

There is no online account support for Google, Microsoft or Drop box. No one uses fast-mail and last.fm. Most of the users have Gmail and Outlook accounts.

## Proposal

I can understand that you guys must be busy in eOS 6 development. Also adding support and integrating these features might not be easy as it sounds. But guys this is my humble request to team elementary to consider adding support to Gdrive , Gmail and Onedrive just like Gnome.

**Comment by corbin-auriti** [comment]
You need to install pantheon-online-accounts-plugin-google pantheon-online-accounts-microsoft but at the moment there is no Dropbox

**Reply by marbetschar** [reply]
FYI: This approach no longer works on OS 6, since we are reworking how Online Accounts works due to issues with Flatpak (see my answer below)

**Comment by kdwk** [comment]
** (io.elementary.switchboard:11560): CRITICAL **: 21:06:37.115: Account.vala:103: GDBus.Error:com.google.code.AccountsSSO.gSingleSignOn.Error.MethodNotKnown: authentication method 'oauth' doesn't exist

@RavetcoFX Seems like pantheon-online-accounts-plugin-google does not work. Also, pantheon-online-accounts-microsoft doesn't seem to exist

**Reply by marbetschar** [reply]
FYI: This approach no longer works on OS 6, since we are reworking how Online Accounts works due to issues with Flatpak (see my answer below)

**Comment by LuisMi1245** [comment]
I don't quite understand the reason for the connection to be through mail and calendar protocols either. The philosophy has always been that the user doesn't have to know how to do something, EOS already does it for us. But if the implementation works this way for now, the developers will have their reasons.

**Comment by marbetschar** [comment]
Yeah, we ran into some issues sharing credentials with Flatpack sandboxed apps (https://github.com/elementary/tasks/issues/269, https://github.com/elementary/tasks/issues/263, https://github.com/elementary/mail/issues/591).

So before we can add more and better Online Account configuration options we need to work this out (https://github.com/elementary/switchboard-plug-onlineaccounts/issues/209).
